open_academy/models/open_academy_course.py

```diff
@@ -7,3 +7,5 @@ class OpenAcademyCourse(models.Model):
 
     title = fields.Char(required=True, size=50)
     description = fields.Text()
+    responsible = fields.Many2one("res.users")
+    sessions = fields.One2many("open.academy.session", "course")
```

Comment:
```suggestion
    responsible_id = fields.Many2one("res.users")
    session_ids = fields.One2many("open.academy.session", "course")
```

See [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#38fields)